src/byron.ts

```diff
@@ -44,9 +44,15 @@ export interface ByronNodeConfig {
   /** Directory containing configurations for all networks. */
   configurationDir: DirPath;
 
+  /** Path to the delegation certificate. */
+  delegationCertificate?: string;
+
   /** Network parameters */
   network: ByronNetwork;
 
+  /** Path to the signing key. */
```

Comment:
Could there be a little more information about what delegation certificate and signing key are, and what they are for?

Reply:
Good catch. I'll see if there's a reference in `cardano-node` to grab

Reply:
Signing key is not described upstream, but I've updated delegation certificate. I'd prefer we stay aligned, so suggest leaving it